backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/projection/impl/ElasticsearchSearchProjectionBuilderFactory.java

```diff
@@ -19,6 +19,7 @@
 import org.hibernate.search.backend.elasticsearch.types.projection.impl.ElasticsearchFieldProjectionBuilderFactory;
 import org.hibernate.search.engine.search.SearchProjection;
 import org.hibernate.search.engine.search.predicate.spi.DslConverter;
+import org.hibernate.search.engine.search.predicate.spi.ProjectionConverter;
```

Comment:
`ProjectionConverter` should be in the `projection` package... `org.hibernate.search.engine.search.projection.spi.ProjectionConverter`

Reply:
you're right! makes sense